### ISSUE TITLE
exclusion patterns

### ISSUE THREAD

**Issue by slipset** [adamtornhill/code-maat#13]
I just started playing with code-maat, and I find I do something like this:
` lein.bat run -l logfile.log -c git | grep -v pom.xml | grep -v test | head`
It would be really nice to be able to specify these exclusions on the command-line of code-maat instead piping to commands

**Comment by adamtornhill** [comment]
Yes, that's a good idea. So far I've done it by shell scripts that clean the output. But it would be possible to do much earlier in the analysis pipe, just after the input log has been parsed.

Do you have any idea for the exclusion pattern format? There's a risk I'll go for regex otherwise.

**Comment by slipset** [comment]
I think I'd be happy with a comma-separated list of ant-style glob patterns like:

`--exclusions='**/test**,**/pom.xml,**/resources/**/*.properties'`

but I'd be happy with some regexes as well.
